docs/development.md

```diff
@@ -100,7 +100,7 @@ Click the "Terminate" icon on the Eclipse console.
 1. If you do not want the app to auto launch on the browser, under `Open browser`, uncheck `After launch`.
 1. Check in `JRE:` that the SDK used is the default of `1.7`.\
    Otherwise your project might not have been configured correctly, please [check your setup](ide-usage.md#prerequisites).
-1. Set the `Port:` to `8888`.
+1. Set the `Port:` to `8080`.
```

Comment:
Can delete this step instead as the default is 8080.

Reply:
If this is the only change required, can defer this to when we're revisiting IntelliJ setup?